components/Order/order_states.rst

```diff
@@ -4,7 +4,7 @@ Order States
 Sylius itself uses a complex state machine system to manage all states of the business domain. 
 This component has some sensible default states defined in the **OrderInterface**.
 
-Default States
+Default states
```

Comment:
You use "Order Events" but here "Default states". I think that you should choose one convention about using lowercase/capital letters.